src/main/java/org/aerogear/digger/client/DiggerClient.java

```diff
@@ -280,6 +282,31 @@ public BuildTriggerStatus build(String jobName, long timeout) throws DiggerClien
     }
 
     /**
+     *
```

Comment:
A one-liner description of this method would be useful. Sth like 
"Triggers a build for the given job. Unlike {@link build(String, long, Map<String, String>)}, this method returns immediately with the build status, which has a queue reference in it to track the build."

Reply:
Agree the doc here should contain info about the purpose of this and how it differs from build